From fb724063b60139468caceec06b946b82235578bc Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekletar@users.noreply.github.com>
Date: Tue, 14 Mar 2017 22:38:19 +0100
Subject: [PATCH] machinectl: don't return 1 in case we couldn't figure out
 container IP addresses (#5587)

This is in spirit very similar to commit
4b2419165ce409ee55ce96a926302f89685f2293.

Fixes: #5581
(cherry picked from commit 3c302cddfb2e921578d1238ebcc0cb5ff34fbebe)
---
 src/machine/machinectl.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/machine/machinectl.c b/src/machine/machinectl.c
index 28384286fb..3031ed5def 100644
--- a/src/machine/machinectl.c
+++ b/src/machine/machinectl.c
@@ -340,6 +340,7 @@ static int list_machines(int argc, char *argv[], void *userdata) {
                         printf("No machines.\n");
         }
 
+        r = 0;
 out:
         clean_machine_info(machines, n_machines);
         return r;
